redis/pool.go

```diff
@@ -185,11 +185,8 @@ func NewPool(newFn func() (Conn, error), maxIdle int) *Pool {
 // getting an underlying connection, then the connection Err, Do, Send, Flush
 // and Receive methods return that error.
 func (p *Pool) Get() Conn {
-	pc, err := p.get(nil)
-	if err != nil {
-		return errorConn{err}
-	}
-	return &activeConn{p: p, pc: pc}
+	c, _ := p.GetContext(context.Background())
```

Comment:
Your discarding the error here, which isn't safe

Reply:
Like `Get`, `GetContext` returns `errorConn` in case of an error, so I think it's okay.

https://github.com/gomodule/redigo/blob/129455dc28ae7cb56466318ec259422d14851207/redis/pool.go#L203-L209

Reply:
We should add a comment to clarify that as its not immediately clear.

Reply:
Added a comment!